### PR TITLE
Operator 1.1 RN

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

https://deploy-preview-5741--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-operator/release-notes/operator-1.1/

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Operator v1.1, detailing new features, enhancements, and upgrade instructions.
- Introduced support for Tyk Streams API through a new custom resource, `TykStreamsApiDefinition`.
- Updated the release notes overview and menu to include links to the new v1.1 release notes.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>operator-1.1.md</strong><dd><code>Add Tyk Operator v1.1 Release Notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-operator/release-notes/operator-1.1.md

<li>Added release notes for Tyk Operator v1.1.<br> <li> Documented support for Tyk Streams API.<br> <li> Included upgrade instructions and download links.<br> <li> Provided a changelog with new features.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5741/files#diff-d103ab72a7e98c900c58f923fbef04fb88543c8ad5b013e04df6aaffc6e7fa0b">+230/-0</a>&nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>overview.md</strong><dd><code>Update Release Notes Overview with v1.1</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-operator/release-notes/overview.md

- Added link to Tyk Operator v1.1 release notes.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5741/files#diff-d8da05fbae7af1f8ae2e7ed25e04a83c0fff93103dc37b210a392b235a17cace">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>menu.yaml</strong><dd><code>Update Menu with Tyk Operator v1.1 Entry</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/data/menu.yaml

- Added menu entry for Tyk Operator v1.1 release notes.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5741/files#diff-a6789c66bfa0660f7fdbe4796002ab4777677bae59a0276689c840100ff0b0b2">+4/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-1.1.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->
+<ul>
+<li>
+<details>
+<summary>TykStreamsApiDefinition: new Custom Resource for Tyk Streams</summary>
+
+The `TykStreamsApiDefinition` custom resource allows you to manage Tyk Streams APIs directly within your Kubernetes environment. This enhancement offers a Kubernetes-native approach to managing Tyk APIs, streamlining operations and ensure single source of truth in Kubernetes.
+
+Learn More: TBD
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-1.1.md]:
```suggestion
[Learn More]({{< ref "api-management/automations#create-a-tykstreamsapidefinition-custom-resource" >}})
```

Add the reference link when the operation documentation is live.

**Review by sharadregoti — CHANGES_REQUESTED** [review]
TODO: Change Release Date When merged.

**Review by sharadregoti — APPROVED** [review]

**Review by komalsukhani — APPROVED** [review]

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-1.1.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+## Support Lifetime
+<!-- Required. replace X.Y with this release and set the correct quarter of the year -->
+Our minor releases are supported until our next minor comes out.
+
+---
+
+## 1.1.0 Release Notes
+
+### Release Date XX Nov 2024 (TBD)
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-1.1.md]:
```suggestion
### Release Date 09 December 2024
```

**Pull request merged**